src/Controller/AjaxController.php

```diff
@@ -49,13 +48,12 @@ public function __construct(CmsManagerSelectorInterface $cmsManagerSelector, Blo
     /**
      * Action for ajax route rendering a block by calling his executeAjax() method.
      *
-     * @param Request $request Symfony request object
-     * @param int     $pageId  Page identifier
-     * @param int     $blockId Block identifier
+     * @param int $pageId  Page identifier
+     * @param int $blockId Block identifier
```

Comment:
int|string I think.

Reply:
Im trying to focus only on needed changes for sf 5/6. This will go to a PHPStan up PR.